lib/ex_doc/formatter/html.ex

```diff
@@ -44,7 +44,7 @@ defmodule ExDoc.Formatter.HTML do
   end
 
   defp normalize_config(%{main: "index"}) do
-    raise ArgumentError, message: "\"main\" cannot be set to \"index\", otherwise it will recursively link to itself"
+    raise ArgumentError, message: ~S("main" cannot be set to "index", otherwise it will recursively link to itself)
```

Comment:
Until now I didn't scape any `(` symbol inside of a `~S` or `~s`. I think that the delimiter usage depends of the context, for example, in some tests you can find `~s[..]`